### PR TITLE
feat: support shorter urls for github repositories

### PR DESCRIPTION
**Pull request by mudler**
This allows to define shorter URLs to `/models/apply`. For instance `github:go-skynet/model-gallery/gpt4all-j.yaml` will automatically resolve to: `https://raw.githubusercontent.com/go-skynet/model-gallery/main/gpt4all-j.yaml`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,6 +65,7 @@ gpt4all:
 	@find ./gpt4all -type f -name "*.cpp" -exec sed -i'' -e 's/json_/json_gptj_/g' {} +
 	@find ./gpt4all -type f -name "*.cpp" -exec sed -i'' -e 's/void replace/void json_gptj_replace/g' {} +
 	@find ./gpt4all -type f -name "*.cpp" -exec sed -i'' -e 's/::replace/::json_gptj_replace/g' {} +
+	@find ./gpt4all -type f -name "*.cpp" -exec sed -i'' -e 's/regex_escape/gpt4allregex_escape/g' {} +
```
Comment (Aisuko) [inline Makefile:68]: Is this one related to this PR?

Comment (mudler) [reply Makefile:68]: Yes, fails otherwise locally to build

Comment (Aisuko) [reply Makefile:68]: Gotcha
